charts/backingservices/README.md

```diff
@@ -31,18 +31,19 @@ If an externally managed elasticsearch cluster is being used, make sure the serv
-`srsStorage.provisionInternalESCluster` | <ul><li>Enable this parameter to provision an internally managed and secured Elasticsearch cluster to be used with the SRS (Requires you to run `$ make es-prerequisite NAMESPACE=<NAMESPACE_USED_FOR_DEPLOYMENT>`).</li><li>Disable this parameter to use your own Elasticsearch service with the SRS.</li></ul>
+| Configuration                           | Usage                                                                                                                                                                                                                                                                                                                                                                     |
+|-----------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `enabled`                               | Enable the Search and Reporting Service deployment as a backing service.                                                                                                                                                                                                                                                                                                  |
+| `deploymentName`                        | Specify the name of your SRS cluster. Your deployment creates resources prefixed with this string. This is also the service name for the SRS.                                                                                                                                                                                                                             |
+| `srsRuntime`                            | Use this section to define specific resource configuration options like image, replica count, cpu and memory resource settings in the SRS.                                                                                                                                                                                                                                |
+| `elasticsearch`                         | Define the elasticsearch cluster configurations. The [Elasticsearch](https://github.com/helm/charts/tree/master/stable/elasticsearch/values.yaml) chart defines the values for elasticsearch provisioning in the cluster. For internally provisioned elasticsearch version is set to 7.10.2. Use imageTag configuration in values.yaml to update it to 7.16.3 if required |
```

Comment:
slight tweaks>>
| `elasticsearch`                         | Define the Elasticsearch cluster configurations. The [Elasticsearch](https://github.com/helm/charts/tree/master/stable/elasticsearch/values.yaml) chart defines the values for Elasticsearch provisioning in the SRS cluster. For internally provisioned Elasticsearch the default version is set to `7.10.2`. Set the `elasticsearch.imageTag` paramater in values.yaml to `7.16.3` to use this supported version in the SRS cluster. |